features/password_reset.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Password reset
+    As a user 
+    If I lose my password
```

Comment:
This is the line I was referring to before. It should be something like: 
As a user
In order to retrieve my password
I would like to be able to....